lib/sidekiq/redis_iterator.rb

```diff
@@ -0,0 +1,16 @@
+module Sidekiq
+  module RedisIterator
```

Comment:
Since it's only used within the API, put in sidekiq/api rather than its own file?  Is `RedisScanner` a better name?